civihr_default_theme/scss/components/_header.scss

```diff
@@ -87,6 +87,10 @@
     display: none;
     height: 50px;
     text-align: center;
+
+    &:hover {
+      color: rgba($light, 0.8);
```

Comment:
This will affect mobile devices also I think. In mobile you dont have any hover event, but the when you tap the element, the hover css gets applied. Could you check if that is the case here.

Reply:
@deb1990 it doesn't, as it targets the logo in the "corner" element, which is shown only on desktop
```scss
.chr_header__corner {
  > .chr_header_logo {
    &:hover {
    }
  }
}